README.rst

```diff
@@ -615,6 +615,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **native_function_casing** [contrib]
```

Comment:
> cannot be tested against by the fixer as it would require knowledge of the whole code base

Isn't this one of original considering all function-replacing fixers as risky?
As we only know what happens in _single_ file?